main/getting-started/intro-zoe.md

```diff
@@ -1,358 +1,332 @@
 # Introduction to Zoe
 
-**Note:** Before reading this, you should be familiar with [ERTP basics](/getting-started/ertp-introduction/). 
-
 ::: tip Alpha status
 Zoe is currently at Alpha status. It has not yet been
 formally tested or hardened. It is not yet of production quality.
 :::
 
-This is an introduction to Zoe, Agoric's smart contract framework. 
-The [Zoe Guide](/zoe/guide/) has additional information, 
-including concepts not covered here. 
+## What is Zoe?
 
-You can run different types of smart contracts on Zoe. In this document, 
-for the most part we focus on Agoric’s Atomic Swap demo contract, but 
-sometimes refer to how Zoe works with other contract types.
+Zoe is Agoric's smart contract framework. You should use Zoe to:
 
-## What is Zoe?
+* **Run your code on-chain**
+* **Mint new digital assets**
+* **Credibly trade assets**
+
+Zoe relies on [ERTP](./ertp-introduction.md), our token standard for fungible and non-fungible
+tokens. 
+
+## Why should I use Zoe?
+
+### For Users ###
+
+**Zoe is safer.** Putting digital assets in a smart contract has
+traditionally meant you might lose them. But Zoe guarantees that you
+will either get what you wanted or get a full refund. You will never
+leave a contract empty-handed, even if the smart contract is buggy or
+malicious. 
+
+### For Developers ###
+
+**Zoe is easier.**  Writing a smart contract has traditionally meant
+learning a completely new, untried language. And let's hope you don't
+make any mistakes - if you do, your users might lose millions.
+
+Zoe smart contracts, on the other hand, are written in a secure subset
+of JavaScript. Moreover, Zoe handles all of the escrowing of digital
+assets provided by your users and the subsequent payout of digital
+assets to your users. **Your contract, even if it's buggy, can't cause
+your users to lose their assets.**
```

Comment:
Just an idea.
```suggestion
assets to your users. **Your contract, even if it has bugs due to a library you 
relied on, can't cause your users to lose their assets.**
```

Reply:
I like this concept, but I think it might be too wordy for this particular sentence. Is there a good way to still include?

Reply:
This is shorter. Does it work here?

"even if you relied on a buggy contract"

Reply:
"even if you relied on a buggy library"

Reply:
I think this is changing the meaning significantly. I'm trying to say, 'even if the contract *you wrote* is buggy, your users are fine" I think bringing up other contracts and libraries is a good point, but it doesn't seem to fit well.

Reply:
My goal was to seem less accusatory. (Some) People don't immediately accept that they need to protect customers from their own mistakes, but I think the argument that code you rely on may have unnoticed bugs or bugs introduced later is becoming more accepted.

Reply:
Hmm, but I think some people aren't relying on any libraries or other code, so that argument doesn't work for them. But I do think that most responsible smart contract devs would be worried about a Parity-like problem.